homeassistant/components/philips_js/__init__.py

```diff
@@ -131,13 +131,13 @@ def system(self) -> SystemType:
         """Return the system descriptor."""
         if self.api.system:
             return self.api.system
+        assert self.config_entry
         return self.config_entry.data[CONF_SYSTEM]
 
     @property
     def unique_id(self) -> str:
         """Return the system descriptor."""
-        entry: ConfigEntry = self.config_entry
-        assert entry
+        assert (entry := self.config_entry) is not None
```

Comment:
Same comment as above, adjust the typing remove the need to assert in this case.